packages/lesswrong/components/sequences/SequencesGridItem.jsx

```diff
@@ -6,68 +6,71 @@ import {
 } from 'meteor/vulcan:core';
 import { Image } from 'cloudinary-react';
 import NoSSR from 'react-no-ssr';
-import React from 'react';
+import React, { PureComponent } from 'react';
 import Users from 'meteor/vulcan:users';
-import { Link } from 'react-router';
+import { Link, withRouter } from 'react-router';
 import classNames from 'classnames';
 import Typography from '@material-ui/core/Typography';
 import { withStyles } from '@material-ui/core/styles';
 
 const styles = theme => ({
   root: {
-    ...theme.typography.postStyle
+    ...theme.typography.postStyle,
+    cursor: 'pointer',
   }
 })
 
-const SequencesGridItem = ({
-  sequence,
-  currentUser,
-  showAuthor=false,
-  classes
-}) => {
-  // const allPostsList = sequence.chapters && _.reduce(sequence.chapters, (memo, c) => [...memo, ...c.posts], []);
-  // const totalPostsNumber = _.reduce(sequence.chapters, (memo, c) => [...memo, ...c.postIds], []).length;
-  // <div className="sequences-grid-item-progress" style={{color: sequence.color}}>{totalPostsNumber} articles</div>
+class SequencesGridItem extends PureComponent {
+  getSequenceUrl = () => {
+    return '/s/' + this.props.sequence._id
+  }
 
-  // const readPostsNumber = allPostsList && _.filter(allPostsList, (p) => p && p.lastVisitedAt).length;
-  const cloudinaryCloudName = getSetting('cloudinary.cloudName', 'lesswrong-2-0')
+  handleClick = (event) => {
+    const { router } = this.props
+    if (!event.target.closest('a')) { // Checks whether any parent is a tag
```

Comment:
There's some duplication between this code and the other place that you're doing the same thing. I suggest making a `withClickNavigation` component which takes a route and children as props. It can add the click handler and render the children.